services/web/server/requirements/constraints.txt

```diff
@@ -14,3 +14,8 @@ pyinstrument<4
 
 # SEE services/web/server/tests/unit/isolated/test_utils.py::test_yarl_url_compose_changed_with_latest_release
 yarl<1.6
+
+
+# 3.9.0 tests fail
+#
+aiohttp<=3.9.0
```

Comment:
I am not sure I understand the comment with the constraint. 3.9.0 fails and you set it to 3.9.0???